src/docs/cookbook/navigation/navigation-basics.md

```diff
@@ -68,18 +71,15 @@ class SecondScreen extends StatelessWidget {
 
 ## 2. Navigate to the second screen using `Navigator.push`
 
-In order to Navigate to a new screen, we'll need to use the
-[`Navigator.push`](https://docs.flutter.io/flutter/widgets/Navigator/push.html)
-method. The `push` method will add a `Route` to the stack of routes managed by
-the Navigator!
+The [`Navigator.push`](https://docs.flutter.io/flutter/widgets/Navigator/push.html) method 
+will **add** a `Route` to the stack of routes managed by the Navigator.
 
 The `push` method requires a `Route`, but where does the `Route` come from?
 We can create our own, or use the [`MaterialPageRoute`](https://docs.flutter.io/flutter/material/MaterialPageRoute-class.html)
```

Comment:
This page really needs an overhaul. Are you sure you want to deal with this?  :D